docs/tools/lsp-factoryjs/classes/02-lsp4-digital-asset-metadata.md

```diff
@@ -0,0 +1,153 @@
+---
+sidebar_position: 2
+title: LSP4DigitalAssetMetadata
+---
+
+# LSP4DigitalAssetMetadata
+
+## uploadMetadata
+
+```js
+LSP4DigitalAssetMetadata.uploadMetadata(lsp4Metadata, uploadOptions?);
+```
+
+The function uploads and processes passed assets and images, and uploads the [LSP4 Digital Asset Metadata](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-4-DigitalAsset-Metadata.md) to IPFS.
+
+The `uploadMetadata` function is available as a static or non-static function callable on the `LSPFactory` library instance. <br/> If `uploadOptions` are not specified in the function call, and the function is used on an `LSPFactory` instance, <br/> the specified options in `uploadOptions` that were passed to the LSPFactory will be used.
```

Comment:
Why take a new line in the middle of a sentence? Can we use regular line breaks instead of `<br/>`, mixing markdown and HTML can get a bit confusing to read

Reply:
`are not specified in the function call, and the function...` -> `are not specified in the function call and the function...`

Normally there is no need for a comma before conjunction words like 'and' 'but' etc.

Reply:
`are not specified in the function call, and the function...` -> `are not specified in the function call and the function...`

Normally there is no need for a comma before conjunction words like 'and' 'but' etc.

Reply:
agree with the HTML, but the comma is there because it is an insertion into the sentence.